### ISSUE TITLE
Failure on Fedora30 (Dyninst 10?)

### ISSUE THREAD

**Issue by loveshack** [bsc-performance-tools/extrae#30]
Building fails on Fedora 30.  It's probably due to Dyninst 10, but I haven't investigated as F30 isn't of interest for using extrae. There's output from the failed build that won't last too long at https://copr-be.cloud.fedoraproject.org/results/loveshack/livhpc/fedora-30-x86_64/00920021-extrae/build.log.gz

**Comment by emercadal** [comment]
Hi Dave, Dyninst 10 changed quite significantly and Extrae isn't on par yet. We are working to have the support updated ASAP. In the meantime please compile Extrae without Dyninst and use the LD_PRELOAD method.

**Comment by loveshack** [comment]
> Hi Dave, Dyninst 10 changed quite significantly and Extrae isn't on
> par yet. We are working to have the support updated ASAP. In the
> meantime please compile Extrae without Dyninst and use the LD_PRELOAD
> method.

Yes, thanks.  I don't actually need it, and I reported it for your
information, but you're obviously a step ahead!

**Comment by hainest** [comment]
I'm able to build the current HEAD of master (c899193d) with Dyninst 13.0.0, so this appears to be fixed. If you have any troubles with Dyninst, please submit an issue at [dyninst/dyninst](https://github.com/dyninst/dyninst/issues) and I'd be happy to help.

@emercadal I think this can be closed.
